### PR TITLE
feat/savax-oracle Adding sAVAX oracle and deployment script for sAVAX vault

### PR DESCRIPTION
**Pull request by fqlx**
feat/savax-oracle Adding sAVAX oracle and deployment script for sAVAX vault

### CODE REVIEW

**Review by kenchangh — APPROVED** [review]
LGTM

**Review by kenchangh — APPROVED** [review]
LGTM

File: .eslintrc.json [hunk 1]
```diff
@@ -49,13 +49,6 @@
         "capitalized-comments": "off",
         "class-methods-use-this": "error",
         "comma-dangle": "off",
-        "comma-spacing": [
```
Comment (kenchangh) [inline .eslintrc.json:52]: Is there a reason why we're removing this lint rule?

Comment (fqlx) [reply .eslintrc.json:52]: It was conflicting with itself during lint fix and prettier.  
`[x, , , , ]` vs `[x, , , ,]`

**Pull request merged**